aws-connect

```diff
@@ -67,7 +68,7 @@ get_instances_by_tag() {
 }
 
 # get the options and set flags
-while getopts "a:d:c:g:w:n:t:r:p:o:x:shv" OPTION; do
+while getopts "a:d:c:l:g:w:n:t:r:p:o:x:shv" OPTION; do
```

Comment:
Make -l a parameter that doesn't require an arg (like -v).  It's either there or not, there's no need to specify `true` or `false` then.